Asterisk.2013/Asterisk.NET/Common.cs

```diff
@@ -26,7 +27,29 @@ public static class Common
         public static char INTERNAL_ACTION_ID_DELIMITER = '#';
 
         /// <summary> Variables delimiter </summary>
-        public static char[] VAR_DELIMITER = {'|'};
+        public static Dictionary<string, char[]> VAR_DELIMITERS = new Dictionary<string, char[]>();
+
+        /// <summary> Variables delimiter getter </summary>
+        public static char[] GET_VAR_DELIMITER(string hostname)
+        {
+            if (!VAR_DELIMITERS.ContainsKey(hostname))
+            {
+                VAR_DELIMITERS.Add(hostname, new char[] { '|' });

```

Comment:
Should it return anything at all by default? If it's not been set by "determineVersion" then something's wrong. Either it's a version we've not explicitly validated in determineVersion, or something else.

Either way, just a thought.